static/js/github-create-branch.js

```diff
@@ -0,0 +1,13 @@
+$(document).ready(() => {
+  $("#ghServers").auiSelect2();
+  $("#ghRepo").auiSelect2();
+  $("#ghParentBranch").auiSelect2();
+
+  $(".gitHubCreateBranch__option").on("click", event => {
```

Comment:
Is this still needed? Couldn't find .gitHubCreateBranch__option? could have just missed it tho lolol

Reply:
Ya missed out this one. This has been removed.